packages/ui/src/fields/Textarea/index.scss

```diff
@@ -8,20 +8,23 @@
 
     textarea {
       @include formInput();
-      field-sizing: content;
       overflow-y: auto;
       resize: vertical;
       min-height: base(3);
-      height: 100%;
+      height: auto;
       display: flex;
     }
 
+    textarea:not(:empty) {
+      field-sizing: content;
+      min-height: calc(var(--rows) * base(1) + base(0.8) + 2px);
```

Comment:
Lets start using `var(--base)` when we can. I think we should stop using scss functions everywhere. So in this case it would be `min-height: calc(var(--rows) * var(--base)) + calc(var(--base) * (0.8)) + 2px;`

Maybe you can wrap the whole thing in a single calc, but honestly unsure.